### DISCUSSION TITLE
Welcome to install_pressticket-phpmyadmin Discussions!

### DISCUSSION THREAD

**Discussion by tonnybarros** [tonnybarros/install_pressticket-phpmyadmin#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexisgec** [comment]
Hi! can you help me with this? in search i have an Internal Server Error, not sure how to debug it..

https://prnt.sc/XzYHnGKH5z2I
https://prnt.sc/YjFpNcWkUOQR

do you have some advice?
